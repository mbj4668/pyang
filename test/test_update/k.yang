module k {
  namespace urn:k;
  prefix k;

  revision 2014-03-01;
}
