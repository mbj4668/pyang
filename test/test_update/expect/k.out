k@2014-04-01.yang:1: warning: WBAD_REVISION
k@2014-04-01.yang:1: error: CHK_BAD_REVISION
